data/data/bootstrap/files/usr/local/bin/installer-gather.sh

```diff
@@ -11,6 +11,10 @@ mkdir -p "${ARTIFACTS}"
 
 exec &> >(tee "${ARTIFACTS}/gather.log")
 
+echo "Gathering bootstrap service records ..."
+mkdir -p "${ARTIFACTS}/bootstrap/services"
+sudo cp -r /var/log/openshift/* "${ARTIFACTS}/bootstrap/services/"
```

Comment:
This does not respect `$SERVICE_RECORDS_DIR` but that seems ok to me.

Reply:
I think that is fine. The `$SERVICE_RECORDS_DIR` override is really just there for testing the entry recording script locally.